src/state-tomo.jl

```diff
@@ -73,7 +73,7 @@ function fit(method::FreeLSStateTomo,
 
     reg = (LinearAlgebra.Diagonal(1 ./ sqrt.(vars)) * method.pred) \ (LinearAlgebra.Diagonal(1 ./ sqrt.(vars)) * means)
     return reshape(reg,d,d),
-           sqrt(dot(method.pred*reg-means,LinearAlgebra.Diagonal(vars)\(method.pred*reg-means)))/length(means),
+           sqrt(LinearAlgebra.dot(method.pred*reg-means,LinearAlgebra.Diagonal(vars)\(method.pred*reg-means)))/length(means),
```

Comment:
This is by choice so it's obvious where the function came from.  The `LinearAlgebra` portion should have been there but was left off.  Looks goo though I'm going to try and write some tests for this in the future.